data-collection/src/test/resources/log4j2.xml

```diff
@@ -1,43 +1,56 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="WARN">
+    <Properties>
+        <Property name="exceptions">false</Property>
+        <Property name="fileSize">20 MB</Property>
+	<Property name="workerName">test</Property>
+        <Property name="filePath">./logs/data-collection_${workerName}</Property>
```

Comment:
Do we need to update the log4j2 in the test folder? I suppose this one will never go through Docker (only local tests + CI)?

Reply:
You are right, It will (probably) never get in touch with docker. Thus, we also don't fetch the docker container name here but replace it with `test`.
I changed this log4j config file in order to have the same structure in both versions (main/ test).